.github/workflows/run_tests.yml

```diff
@@ -100,7 +100,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: "3.7"
+          python-version: "3.11"
```

Comment:
There's a bug in coverage which is triggered when coverage data is captured on 3.7 or earlier but the report is generated on 3.8 or later. See e9f0e5d5 for the previous change which fixed this. I think this change needs to be reverted to maintain coverage reporting on Windows, which is currently only running tests on Python 3.7 due to other issues.

Reply:
Reverted